cmd/offat/main.go

```diff
@@ -103,6 +106,8 @@ func main() {
 		log.Error().Err(err).Msg("failed to set baseUrl")
 	}
 
+	fmt.Println(config.BaseUrl)
```

Comment:
we can remove debugging statements

Reply:
Sorry for that. I will make sure to get rid of the debugging statements.